src/components/editor/faultTree/menu/faultEvent/FaultEventMenu.tsx

```diff
@@ -376,37 +375,37 @@ const FaultEventMenu = ({ shapeToolData, onEventUpdated, refreshTree, rootIri }:
             {criticality && (
               <Typography>
                 <span className={classes.label}>{t("faultEventMenu.criticality")}:</span>
-                <span className={classes.grey}>{criticality}</span>
+                <span className={classes.selected}>{criticality}</span>
               </Typography>
             )}
             {ataSystem && (
               <Typography>
                 <span className={classes.label}>{t("faultEventMenu.ataSystem")}:</span>
-                <span className={classes.grey}>{ataSystem}</span>
+                <span className={classes.selected}>{ataSystem}</span>
```

Comment:
see above and search for multiple references like that

Reply:
Fixed as above